app/models/search.rb

```diff
@@ -3,7 +3,7 @@
 class Search < ApplicationRecord
   belongs_to :user, optional: true
 
-  serialize :query_params
+  serialize :query_params, Blacklight::SearchParamsYamlCoder
```

Comment:
Maybe a comment explaining why this is necessary, and that it becomes no longer necessary if in a Rails version including https://github.com/rails/rails/pull/45591

(If that rails gets merged, as it looks like it will, then at some future point where BL only supports Rails versions where the latest patch includes that rails change, this custom coder will no longer be necessary)